### PR TITLE
New module gmute & score

### PR DESCRIPTION
**Pull request by AbhiTheModder**
Hello i got this script from Ultroid team please check if this script can run in dragon too if not then i've to write a new script and if worked then my work became easy 😅

### CODE REVIEW

**Comment by john-phonk** [comment]
**No, this module will not work in dragon**

Dragon custom module example:

```python3
from pyrogram import Client, filters
from pyrogram.types import Message
from utils.misc import modules_help, requirements_list, prefix


# packages from PyPI
# import example_1
# import example_2


@Client.on_message(filters.command("example_edit", prefix) & filters.me)
async def example_edit(client: Client, message: Message):
    await message.edit("<code>This is an example module</code>")


@Client.on_message(filters.command("example_send", prefix) & filters.me)
async def example_send(client: Client, message: Message):
    await client.send_message(message.chat.id, "<b>This is an example module</b>")


# This adds instructions for your module
modules_help.append(
    {"example": [{"example_send": "example send"}, {"example_edit": "example edit"}]}
)

# If your custom module requires packages from PyPI, write the names of the packages in these functions
# requirements_list.append('example_1')
# requirements_list.append('example_2')
# etc
```

**Comment by AbhiTheModder** [comment]
Okay then i 've to write my own script

**Comment by AbhiTheModder** [comment]
So one question also if i make a script how i could check that before creating pr and disturbing the team ?

**Comment by john-phonk** [comment]
Place your module in modules/custom_modules directory and restart userbot

**Comment by AbhiTheModder** [comment]
OKay thanks very much

**Review by iamnalinor — CHANGES_REQUESTED** [review]

File: exceptions.py [hunk 1]
```diff
@@ -0,0 +1,73 @@
+from pyrogram.types import Message, User
```
[the remark this answers is not in the document]
Comment (AbhiTheModder) [reply exceptions.py:1]: okay adding everything from exceptions to gban file

File: exceptions.py [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+    if SELF.status not in ranks:
+        await message.edit("__I'm not Admin!__")
+        sleep(2)
+        await message.delete()
+
+    else:
+        if SELF.status is not admin or SELF.can_restrict_members:
+            return True
+        else:
+            await message.edit("__No Permissions to restrict Members__")
```
[the remark this answers is not in the document]
Comment (AbhiTheModder) [reply exceptions.py:71]: okay

File: exceptions.py [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+    
+def get_arg(message):
+    msg = message.text
+    msg = msg.replace(" ", "", 1) if msg[1] == " " else msg
+    split = msg[1:].replace("\n", " \n").split(" ")
+    if " ".join(split[1:]).strip() == "":
+        return ""
+    return " ".join(split[1:])
+    
+async def CheckAdmin(message: Message):
```
[the remark this answers is not in the document]
Comment (AbhiTheModder) [reply exceptions.py:52]: not understood this can you explain a bit

File: exceptions.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (AbhiTheModder) [reply exceptions.py:1]: i'm confused fully now i'll try tomorrow again and will tell when added things from exceptions it given `name must an instance of str`

[the remark this answers is not in the document]
Comment (AbhiTheModder) [reply exceptions.py:1]: hello i've moved exception to custom_module now itll take things from exceptions.py custom_module which user need install first as moving those thigs inside one file giving me errors

[the remark this answers is not in the document]
Comment (AbhiTheModder) [reply exceptions.py:1]: and this module is working very perfectly i checked 5 times

**Pull request merged**